Code Review Assignment_JrSDET .java

```diff
@@ -21,7 +21,7 @@ public static void main(String[] args) throws InterruptedException, AWTException
 		driver.get("https://interview.supporthive.com/staff/");
 		driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
 		driver.manage().window().maximize();
-		driver.findElement(By.id("id_username")).sendKeys("Agent");
+		driver.findElement(By.id("id_username")).sendKeys("Agent");  //code review

```

Comment:
Web elements should not be directly interacted with in this manner. Instead, they should be stored in variables of the WebElement type using the @FindBy annotation at the beginning of the class. This practice enhances code readability and maintainability